### PR TITLE
cor/mosaic minimum amount out

### PR DESCRIPTION
**Pull request by cor**
new PR bc bors got broken

### CODE REVIEW

**Review by PoisonPhang — APPROVED** [review]

**Review by benluelo — APPROVED** [review]

**Review by KaiserKarel — APPROVED** [review]

**Comment by cor** [comment]
please work bors, i'm begging you

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
 		"rust-analyzer.rustfmt.enableRangeFormatting": true,
 		"[rust]": {
 			"editor.defaultFormatter": "rust-lang.rust-analyzer",
-			"editor.formatOnSave": true
+			"editor.formatOnSave": false
```
Comment (dzmitry-lahoda) [inline .devcontainer/devcontainer.json:42]: not sure I aggree. are you 100% that extensions fromatter is 100% same as CI one?

File: frame/mosaic/src/lib.rs [hunk 2]
```diff
@@ -294,7 +305,8 @@ pub mod pallet {
 			amount: BalanceOf<T>,
 			swap_to_native: bool,
 			source_user_account: AccountIdOf<T>,
-			amm_swap_info: Option<AmmSwapInfo<NetworkIdOf<T>, RemoteAmmIdOf<T>>>,
+			amm_swap_info:
```
Comment (dzmitry-lahoda) [inline frame/mosaic/src/lib.rs:308]: not sure I see it is used anywhere. so adding is good, but make be make TODO or reference click up for later actual usage?

**Review by KaiserKarel — APPROVED** [review]

**Comment by KaiserKarel** [comment]
bors ping

**Comment by KaiserKarel** [comment]
Bors r+

**Review by KaiserKarel — APPROVED** [review]

**Pull request closed without merging**